src/Akeneo/Pim/Enrichment/Component/Product/Updater/Clearer/Field/QuantifiedAssociationFieldClearer.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Pim\Enrichment\Component\Product\Updater\Clearer\Field;
+
+use Akeneo\Pim\Enrichment\Component\Product\Model\EntityWithQuantifiedAssociationsInterface;
+use Akeneo\Pim\Enrichment\Component\Product\Model\QuantifiedAssociation\QuantifiedAssociations;
+use Akeneo\Pim\Enrichment\Component\Product\Updater\Clearer\ClearerInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @author    Julien Sanchez <julien@akeneo.com>
+ * @copyright 2020 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+final class QuantifiedAssociationFieldClearer implements ClearerInterface
+{
+    private const SUPPORTED_FIELD = 'quantified_associations';
+
+    /**
+     * {@inheritDoc}
+     */
+    public function supportsProperty(string $property): bool
+    {
+        return static::SUPPORTED_FIELD === $property;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function clear($entity, string $property, array $options = []): void
+    {
+        Assert::true(
+            $this->supportsProperty($property),
+            sprintf('The clearer does not handle the "%s" property.', $property)
+        );
+
+        if ($entity instanceof EntityWithQuantifiedAssociationsInterface) {
+            $entity->setQuantifiedAssociations(QuantifiedAssociations::createFromNormalized([]));
```

Comment:
To make it work we should fill all association types with empty array because src/Akeneo/Pim/Enrichment/Component/Product/Updater/Setter/QuantifiedAssociationsFieldSetter.php merge by key (maybe the problem is here, we implement it here to handle partial update from API). 

We could handle it with something like that :
```suggestion
$quantifiedAssociationCleared = array_map(function($quantifiedAssociationNormalized) {
    return [];
}, $entity->getQuantifiedAssociations()->normalize());

$entity->setQuantifiedAssociations(QuantifiedAssociations::createFromNormalized($quantifiedAssociationCleared));
```

But I think we can create a ticket to handle it and test it separately